DataFormats/FEDRawData/interface/FEDNumbering.h

```diff
@@ -44,6 +44,8 @@ class FEDNumbering {
      MAXTotemTriggerFEDID = 577,
      MINTotemRPFEDID = 578,
      MAXTotemRPFEDID = 581,
+     MINCTPPSDiamondFEDID=582,
+     MAXCTPPSDiamondFEDID=583,
```

Comment:
9X version has ``` MAXCTPPSDiamondFEDID=585 ``` 
should we keep in sync or is it larger in 9X to cover more detectors not present in 2016?